docs/README.md

```diff
@@ -1330,43 +1330,135 @@ Read Models are classes decorated with the `@ReadModel` decorator that have one
 
 Read Model classes can also be created by hand and there are no restrictions regarding the place you put the files. The structure of the data is totally open and can be as complex as you can manage in your projection functions.
 
 #### The projection function
 A `Projection` is a method decorated with the `@Projects` decorator that, given a new entity value and (optionally) the current read model state, generate a new read model value.
 
-Read models can be projected from multiple [entities](#4-entities-and-reducers) as soon as they share some common key called `joinKey`.
+Read models can be projected from multiple [entities](#4-entities-and-reducers) as soon as they share some common key called `joinKey`. The `joinKey` is the field used across entities to relaunch the projection function. To clarify this let's see an example:
+
+```typescript
+@Projects(Cart, "id")
+	public static projectCart(entity:Cart, currentReadModel: CartReadModel): CartReadModel {
+		return new CartReadModel(entity.id, entity.items)
+	}
+```
+
+In the previous code, the `id` property is used as `joinKey`. This means that each time that a `Cart` that has the same `id` as the read model's `id` is modified, the projection is called.
 
 #### Authorizing read models
 
+Read models are part of the public API of a Booster application, so you can define who is authorized to query them. The Booster authorization feature is covered in [this](https://github.com/boostercloud/booster/blob/master/docs/README.md#authentication-and-authorization) section. So far, we have seen that you can make a read command publicly accessible by authorizing `'all'` to query it. You can also set specific roles providing the users role array in this way: `authorize: [Admin]`.
+
 #### Querying a read model
-You can use the GraphQL endpoint to query or subscribe to the read model records: [see the API documentation](#GraphQL-API).
+
+Read models are the read API for Booster applications. they can be accessed using the GraphQL queries. When you create a read model Booster automatically creates all the neccesary queries for your read models, e.g., given this `CartReadModel` read model:
+
+```typescript
+@ReadModel({
+	authorize: 'all'
+})
+export class CartReadModel {
+	public constructor(
+		public id: UUID,
+		readonly items: Array<CartItem>,
+	) {}
+
+	@Projects(Cart, "id")
+	public static projectCart(entity:Cart, currentReadModel: CartReadModel): CartReadModel {
+		return new CartReadModel(entity.id, entity.items)
+	}
+}
+```
+
+if you hit the provided Url at the end of the deploy process:
+
+```
+<httpURL>/graphql
+```
+
+and you also provide this query using any graphQL client:
+
+```graphQL
+query {
+	CartReadModel(id: "012345") {
+		id
+		items
+	}
+}
+```
+
+you can get some results in the following format:
+
+```json
+{
+	data: {
+		"CartReadModel": {
+			"id" : "012345",
+			"items": [
+				{
+					"sku":"ABC_01",
+					"quantity": 1
+				}
+			]
+		}
+	}
+}
+```
```

Comment:
As this is constantly changing now, I would use the same approach we followed when explaining the same sections for commands, and only talk about one of the queries that are generated. I would not say anything about the URL, and the format of the response. For that, I just refer to the GraphQL API section.

In this case, the query that is generated is: 
```graphql
query CartReadModel(id: ID!): CartReadModel
```

Then I would just say something like "For more information about all the "queries" and how to use them, please go to section (GraphQL API section)"

Reply:
I made this section generic just applying @alvaroloes comment's